roadmap/implementers-guide/src/runtime/disputes.md

```diff
@@ -17,22 +17,15 @@ We account for these requirements by having the disputes module handle two kinds
 1. Local disputes: those contesting the validity of the current fork by disputing a parablock included within it.
 1. Remote disputes: a dispute that has partially or fully resolved on another fork which is transplanted to the local fork for completion and eventual slashing.
 
-When handling disputes on one chain,
+When a local dispute concludes negatively, the chain needs to be abandoned and reverted back to a block where the state does not contain the bad parablock. We expect that due to the [Approval Checking Protocol](../protocol-approval.md), the current executing block should not be finalized. So we do two things when a local dispute concludes negatively:
+1. Freeze the state of parachains so nothing further is backed or included.
+1. Issue a digest in the header of the block that signals to nodes that this branch of the chain is to be abandoned.
```

Comment:
I suppose this points back to the last ancestor that did not include/run the bad parablock.

Reply:
Yep!